_migration-assistant/deploying-migration-assistant/getting-started-data-migration.md

```diff
@@ -145,10 +145,16 @@ Use the following steps to configure and deploy RFS, deploy Migration Assistant,
             }
         },
         "reindexFromSnapshotExtraArgs": "<RFS PARAMETERS (see below)>",
+        "reindexFromSnapshotServiceEnabled": true,
+        "reindexFromSnapshotMaxShardSizeGiB": 80,
```

Comment:
👍 I think this could be useful to leave in since it makes obvious what the max shard size is, and if the user runs into trouble this might help them fix it.